<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDockWidgetVideoRecorder</class>
 <widget class="QDockWidget" name="NewDockWidgetVideoRecorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>416</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Video Recorder</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_19">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabVideoWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabVideoMethod1">
       <attribute name="title">
        <string>Create Video</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_34">
          <item row="0" column="0">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_37">
             <item row="0" column="0">
              <widget class="QTabWidget" name="tabWidget">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="tab">
                <attribute name="title">
                 <string>Fixed Time Point</string>
                </attribute>
                <layout class="QGridLayout" name="gridLayout_6">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_33">
                   <item row="0" column="0">
                    <layout class="QGridLayout" name="gridLayout_18">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_20">
                       <property name="text">
                        <string>T Time </string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QSpinBox" name="tSpinFixed"/>
                     </item>
                    </layout>
                   </item>
                   <item row="1" column="0">
                    <layout class="QGridLayout" name="gridLayout_5">
                     <item row="0" column="1">
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>from</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="2">
                      <widget class="QSpinBox" name="tSpinMin_2"/>
                     </item>
                     <item row="0" column="3">
                      <widget class="QLabel" name="label_7">
                       <property name="text">
                        <string>to</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="4">
                      <widget class="QSpinBox" name="tSpinMax_2"/>
                     </item>
                     <item row="0" column="0">
                      <widget class="QComboBox" name="SliceFT">
                       <item>
                        <property name="text">
                         <string>X Slice</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Y Slice</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Z Slice</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_2">
                <attribute name="title">
                 <string>Through Time Points</string>
                </attribute>
                <layout class="QGridLayout" name="gridLayout_12">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_7">
                   <item row="0" column="0">
                    <layout class="QGridLayout" name="gridLayout_3">
                     <item row="0" column="0">
                      <widget class="QComboBox" name="comboBox_2">
                       <item>
                        <property name="text">
                         <string>X Slice</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Y Slice</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Z Slice</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                     <item row="0" column="3">
                      <widget class="QSpinBox" name="zSpinFixed"/>
                     </item>
                    </layout>
                   </item>
                   <item row="1" column="0">
                    <layout class="QGridLayout" name="gridLayout_8">
                     <item row="0" column="3">
                      <widget class="QSpinBox" name="tSpinMax"/>
                     </item>
                     <item row="0" column="2">
                      <widget class="QLabel" name="label_8">
                       <property name="text">
                        <string>to</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QSpinBox" name="tSpinMin"/>
                     </item>
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>T Time from </string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_14">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="0" column="1">
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="QLineEdit" name="videoName">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="inputMask">
                    <string/>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QPushButton" name="createFile">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>4</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="contextMenuPolicy">
                    <enum>Qt::DefaultContextMenu</enum>
                   </property>
                   <property name="text">
                    <string>Browse</string>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>10</width>
                     <height>10</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="1">
                <layout class="QGridLayout" name="gridLayout_11">
                 <item row="0" column="0">
                  <widget class="QSpinBox" name="frameRate">
                   <property name="value">
                    <number>10</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="2" column="1">
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="0">
                  <widget class="QSpinBox" name="videoQuality">
                   <property name="minimum">
                    <number>0</number>
                   </property>
                   <property name="maximum">
                    <number>2</number>
                   </property>
                   <property name="value">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Name</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Frame Rate</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Video Quality</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="startVideo">
            <property name="text">
             <string>Create video</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabVideoMethod2">
       <attribute name="title">
        <string>Record Video</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_20">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_21">
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_26">
             <item row="0" column="1">
              <widget class="QLineEdit" name="videoName_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QPushButton" name="createFile_2">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>4</width>
                 <height>0</height>
                </size>
               </property>
               <property name="contextMenuPolicy">
                <enum>Qt::DefaultContextMenu</enum>
               </property>
               <property name="text">
                <string>Browse</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>10</width>
                 <height>10</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QGridLayout" name="gridLayout_27">
             <item row="0" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Frame Rate</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="frameRate_2">
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QGridLayout" name="gridLayout_28">
             <item row="0" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Video Quality</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="videoQuality_2">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>2</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_29">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_25">
            <item row="0" column="0">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="startRecord">
              <property name="text">
               <string>Start recording</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_24">
            <item row="0" column="1">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Video lenght (seconds)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLCDNumber" name="videoLenght">
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="horizontalSpacer_17">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_23">
            <item row="0" column="0">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="endRecord">
              <property name="text">
               <string>End recording</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>362</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
